tabular/src/autogluon/tabular/task/tabular_prediction/tabular_prediction.py

```diff
@@ -603,7 +606,10 @@ def fit(train_data,
         if _feature_generator_kwargs:
             if 'feature_generator' in kwargs:
                 logger.log(30, "WARNING: `feature_generator` was specified and will override any presets that alter feature generation (such as 'ignore_text')")
-        feature_generator = kwargs.get('feature_generator', AutoMLPipelineFeatureGenerator(**_feature_generator_kwargs))
+        if 'TEXT_NN_V1' in hyperparameters:
+            _feature_generator_kwargs['enable_raw_text_features'] = True
```

Comment:
@jwmueller @Innixma Here's the logic for setting the `enable_raw_text_features` flag.

Reply:
So I assume that for every new argument we will need to touch the tabular `hyperparameters`? My concern is that will this become extremely hard to maintain in the long run?

Reply:
This will only be temporary. Likely I will focus on smarter logic as this solution is not comprehensive (It doesn't work on the complex hyperparameters format that incorporates level-wise models).

Because it will be temporary, can you add a clear TODO comment regarding this to replace prior to v0.1 release?